docs/oauth2.rst

```diff
@@ -115,12 +115,18 @@ For End Users: Using OAuth Client ID
 This is the case where your application or a script is accessing spreadsheets on behalf of an end user. When you use this scenario, your application or a script will ask the end user (or yourself if you're running it) to grant access to the user's data.
 
 1. :ref:`enable-api-access` if you haven't done it yet.
-2. Go to "APIs & Services > OAuth Consent Screen." Click the button for "Configure Consent Screen" and follow the directions to give your app a name; you don't need to fill out anything else on that screen. Click Save. 
+#. Go to "APIs & Services > OAuth Consent Screen." Click the button for "Configure Consent Screen".
+
+  a. In the "1 OAuth consent screen" tab, give your app a name and fill the "User support email" and "Developer contact information". Click "SAVE AND CONTINUE".
```

Comment:
I beileve this one too, as long as you start your list with a longer indent it will be rendered as an inner-list.

Reply:
Yes, that's the intention, make the points `a` to `d` internal of the step `2.`, since all are steps to do under the "Configure Consent Screen" menu.

Reply:
Yep like above response, the `#` will auto-increment so it's all good.